includes/class-newspack-popups.php

```diff
@@ -261,5 +269,14 @@ public static function display_post_states( $post_states, $post ) {
 		}
 		return $post_states;
 	}
+
+	/**
+	 * Get previewed popup id from the URL.
+	 *
+	 * @return number|null Popup id, if found in the URL
+	 */
+	public static function previewed_popup_id() {
+		return isset($_REQUEST[ Newspack_Popups::NEWSPACK_POPUP_PREVIEW_QUERY_PARAM ]) ? $_REQUEST[ Newspack_Popups::NEWSPACK_POPUP_PREVIEW_QUERY_PARAM ] : NULL;
```

Comment:
This line raises a bunch of PHPCS warnings and errors:

```
phpcs: error
error - Must use "self::" for local static member reference (Squiz.Classes.SelfMemberReference.NotUsed)

phpcs: error
error - Must use "self::" for local static member reference (Squiz.Classes.SelfMemberReference.NotUsed)

phpcs: error
error - TRUE, FALSE and NULL must be lowercase; expected "null" but found "NULL" (Generic.PHP.LowerCaseConstant.Found)

phpcs: warning
warning - Processing form data without nonce verification. (WordPress.Security.NonceVerification.Recommended)

phpcs: warning
warning - Processing form data without nonce verification. (WordPress.Security.NonceVerification.Recommended)

phpcs: warning
warning - Detected usage of a non-sanitized input variable: $_REQUEST[Newspack_Popups::NEWSPACK_POPUP_PREVIEW_QUERY_PARAM] (WordPress.Security.ValidatedSanitizedInput.InputNotSanitized)
```

Suggested alternative:

```suggestion
		return filter_input( INPUT_GET, self::NEWSPACK_POPUP_PREVIEW_QUERY_PARAM, FILTER_SANITIZE_STRING );
```